caddyhttp/httpserver/replacer_test.go

```diff
@@ -473,7 +473,7 @@ func TestRound(t *testing.T) {
 	}
 }
 
-func TestMillisecondConverstion(t *testing.T) {
+func TestMillisecondConversion(t *testing.T) {
```

Comment:
I dont see a corresponding change in the spelling of the use of this function anywhere in your PR but maybe I've missed it.  Do the tests pass.

Reply:
Testing frameworks in fancy languages often use reflection to determine what to run, so this isn't terribly surprising to me for Go/Python/Ruby/JavaScript/Java/Boost C++.